mne/viz/evoked.py

```diff
@@ -1024,6 +1024,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:
+        evoked.pick_types(eeg=True, eog=True, seeg=True, meg=True)
```

Comment:
it's weird I agree but I know that @jona-sassenhagen treats eog as eeg. How about picking channels that have a location?